service/gcsutils/gcstools/commoncli/common.go

```diff
@@ -79,6 +80,15 @@ func SetupLogging(args ...*string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("Invalid log params")
 	}
-	utils.SetLoggingOptions("verbose", *args[0])
+
+	logrus.AddHook(commonutils.NewStackHook([]logrus.Level{logrus.InfoLevel, logrus.DebugLevel}))
```

Comment:
Yeah, you're right, it wasn't configurable before :(. I'm not sure your thoughts, John, but I'm cool with not specifying any stack hooks by default, and just having main.go handle that if specified.